domain/lib/src/model/autocomplete/autocomplete_result_type.dart

```diff
@@ -31,7 +31,7 @@
 //
 
 enum AutoCompleteResultType {
-  simple, user, mailingList
+  simple, user, mailingList, threadmember
```

Comment:
You should use lowerCamelCase `threadMember`

Reply:
it is type from response, so it is no problem with that